.travis.yml

```diff
@@ -5,15 +5,15 @@ node_js:
 env:
   global:
     - secure: dn0FPQ5IG4M/3kdwnyI78ElQ308Vc3QnKAvkWfwMFb8QxDqxQdnTo7AV1qTMtbLrDNkeEWIgi4nc7jmXNtvGTwOfhAULVh6606Qs5B+ezTdwzajbbFMI8SKQx/pnTojOMu8dx7V4lMoR/YWcojR0VC1IWVC62TGbSB1k5BDGgH0=
-    - DOCKER_FOLDER: "https://github.com/openworm/org.geppetto.git#development:utilities/docker/geppetto/"
+    - DOCKER_FOLDER: "https://github.com/openworm/org.geppetto.git#feature/254:utilities/docker/geppetto/"
```

Comment:
Can we target development? From what I understand [the differences between the two](https://github.com/openworm/org.geppetto/compare/development...feature/254) were just for debugging

Reply:
Yes, just reverted this. I expect the last commit with the branch pointing to development to fail in travis, but that's expected.